test/python/circuit/library/test_evolution_gate.py

```diff
@@ -419,10 +420,9 @@ def test_paramtrized_op_raises(self):
         with self.assertRaises(ValueError):
             _ = PauliEvolutionGate(SparsePauliOp("Z", np.array(Parameter("t"))))
 
-    @data(LieTrotter, MatrixExponential)
-    def test_inverse(self, synth_cls):
-        """Test calculating the inverse is correct."""
-        evo = PauliEvolutionGate(X + Y, time=0.12, synthesis=synth_cls())
+    def test_inverse(self):
```

Comment:
should there also be a test for LieTrotter up to some tolerance?

Reply:
should there also be a test for power? control should be covered by test_ctrl_state below?